woocommerce/woocommerce-gutenberg-products-block
# Product Gallery > Next/Previous Buttons block: Add support to Interactivity API

## What
This PR adds support to the Interactivity API on the Next/Previous buttons block, allowing shoppers to navigate between the gallery images without having to reload the page.

Fixes #10631 

## Why
This is part of our effort to implement the new Product Gallery block.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Two," "Twenty-twenty Three," etc.
2. On the left-hand side menu, click on Appearance > Editor > Templates
3. Find and select the 'Single Product' template from the list.
4. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
5. In the block library, search for the 'Product Gallery' block. Click on it to add the block to the template.
6. Check that the Previous/Next button block is displayed on the Editor;
7. Click on save and visit a product page that contains multiple gallery images, make sure the block is correctly being displayed;
8. Make sure you can navigate between the gallery images using the Previous and Next buttons.


## Screenshots or screencast

https://github.com/woocommerce/woocommerce-blocks/assets/20469356/afbb167f-88fe-4084-b29b-33fd0f34511b

## WooCommerce Visibility


Required:
